karton/dashboard/app.py

```diff
@@ -174,6 +175,13 @@ def get_xrefs(root_uid) -> List[Tuple[str, str]]:
 karton_logs = Gauge("karton_logs", "Pending logs")
 karton_tasks = Gauge("karton_tasks", "Pending tasks", ("name", "priority", "status"))
 karton_replicas = Gauge("karton_replicas", "Replicas", ("name", "version"))
+karton_metrics = Gauge("karton_metrics", "Metrics", ("metric", "name"))
+
+
+def add_metrics(state: KartonState, metric: KartonMetrics, key: str) -> None:
+    metrics = {k: int(v) for k, v in state.backend.redis.hgetall(metric.value).items()}
```

Comment:
It would be nice to add this to `karton.backend` as `KartonBackend.get_metrics` function. But this looks ok as well :+1:

Reply:
already done with https://github.com/CERT-Polska/karton/pull/109! but it'll have to wait for a version bum